jetty-http/src/test/java/org/eclipse/jetty/http/HttpParserTest.java

```diff
@@ -714,7 +780,7 @@ public void testCaseInsensitiveHeader() throws Exception
                         "cOnNeCtIoN: ClOsE\r\n" +
                         "\r\n");
         HttpParser.RequestHandler handler = new Handler();
-        HttpParser parser = new HttpParser(handler, -1, HttpCompliance.RFC7230);
+        HttpParser parser = new HttpParser(handler, -1, HttpCompliance.RFC7230_LEGACY);
```

Comment:
We need to keep the older RFC7230 tests too, to ensure we didn't break backward compliance with older 9.4.x releases.

Reply:
Perhaps these tests should be parameterized?

Reply:
Perhaps, but at this stage I'd like to keep the tests pretty much as is, so that we don't inadvertently  change behaviour (but perhaps a few little advertent changes :)